proc_Hahn_echoph.py

```diff
@@ -1,87 +1,69 @@
 from pyspecdata import *
 from scipy.optimize import leastsq,minimize,basinhopping
-fl = figlist_var()
+from proc_scripts import *
+from proc_scripts.load_data import postproc_dict
+from sympy import symbols
+fl = fl_mod()
 t2 = symbols('t2')
 filter_bandwidth = 5e3
-for date,id_string,label_str in [
-        ('191007','echo_2','n'),
+slice_f = (-5e3,5e3)
```

Comment:
slice_f should be associated with the particular experiment (in the list of tuples, next), while filter_bandwidth is not used?

Reply:
deleted filter_bandwidth and slice_f is yes particular to the experiment but most average around here anyhow